packages/alfa-rules/src/sia-r68/rule.ts

```diff
@@ -97,3 +92,37 @@ function isRequiredChild(
     };
   }
 }
+
+/**
+ * Collect all descendants of the given node where the descendant:
+ *
+ * - is a non-ignored HTML or SVG element with a role requiring specific
+ *   children; and
+ * - does not have an `aria-busy` ancestor.
+ */
+function* visit(node: Node, device: Device): Iterable<Element> {
+  if (
+    Refinement.and(isElement, hasAttribute("aria-busy", equals("true")))(node)
```

Comment:
`Refinement.and()` falls back to the signature of `Predicate.and()` when no refinements are involved. You can therefore pull `and()` from `Refinement` rather than `Predicate` to avoid using both.

Reply:
Defaulted to `Refinement` for all logical connectors as it seems easier. But not sure it really is the correct thing to do (vs pulling `and` from `Refinement` and `equals, or` from `Predicate`…)

Reply:
They _all_ fall back to the equivalent `Predicate` signature so that's fine 👍